content/cucumber/state.md

```diff
@@ -9,38 +9,41 @@ title: State
 
 # State
 
-It's important to prevent state created by one scenario to leak into the next.
-Leaking state makes your scenarios brittle and very difficult to run in isolation.
+It's important to prevent state created by one Scenario from leaking into others.
+Leaking state makes your Scenarios brittle, and difficult to run in isolation.
 
 Avoid using global or static variables.
 
-Make sure you clean your database between scenarios.
+Make sure you clean your database between Scenarios.
 
 ## Databases
 
-The recommended approach to clean a database between scenarios is to use a
-[[Before Hook]] to remove all data *before* a scenario starts.
+### The Before Hook Approach
 
-This is usually better than using an [[After Hook]] as it allows
-you to perform a post-mortem inspection of the database in case a scenario fails.
+The recommended approach to clean a database between Scenarios is to use a
+[[Before Hook]] to remove all data *before* a Scenario starts.
+
+This is usually better than using an [[After Hook]], as it allows
+you to perform a post-mortem inspection of the database if a Scenario fails.
 
 An alternative approach is to use database transactions.
 
-## Transactions
+### The Database Transaction Approach
 
 You can wrap a transaction (if your database supports it) *around* each Scenario.
 
-(This might lead to faster scenarios, but it comes at a cost. You won't be able
-to perform a post-mortem, and you won't be able to use [browser automation](/cucumber/browser-automation/)).
+(This might lead to faster Scenarios, but it comes at a cost. 
+You won't be able to perform a post-mortem, and you won't be able to 
+use [browser automation](/cucumber/browser-automation/)).
 
-You simply tell Cucumber to start a transaction in a [[Before Hook]] and roll it back
-in an [[After Hook]].
+You simply tell Cucumber to start a transaction in a [[Before Hook]], and later 
+roll it back in an [[After Hook]].
 
 This is such a common thing to do that several Cucumber extensions provide ready-to-use
 [[Tagged Hooks]] using a tag named `@txn`.
 
-To enable it you have to tag every [[Feature]] or [[Scenario]]
-that needs transactions with `@txn`:
+To enable it, you must to tag every [[Feature]] or [[Scenario]] that requires 
```

Comment:
Remove 'to' from:  'must *to* tag every' (typo as a result of the edit)

Reply:
Thanks!  Fixed.